recipes/gams-ac

```diff
@@ -0,0 +1,5 @@
+(gams-ac
+ :repo "ShiroTakeda/gams-ac"
+ :fetcher github
+ :files ("gams-ac.el")
```

Comment:
please remove the `:files` line, it is [probably useless](https://github.com/melpa/melpa#recipe-format).